### DISCUSSION TITLE
Bots

### DISCUSSION THREAD

**Discussion by draew6** [Byloth/cmangos-docker#32] [General]
Hi, I really like what you did with this repo. This is first time i was able to run server because of your very precise instructions. What i still struggle is to put bots on server. Do you have instructions how to do that easiest way? I saw your bot branch but i doesnt look like it will be soon merged/finished. Thank you

**Comment by Byloth** [comment]
Hi, @draew6!
Thanks for your kind words and your interest into this project! 🙃

Actually... Bots aren't quite yet supported due to some problems on the original CMaNGOS repository.  
More specifically, bots aren't supported on the `tbc` version of the server.

Here some further information about it: https://github.com/Byloth/cmangos-docker/issues/26

---

Once a while, I still trying to make it work to figure out if they have fixed this issue or not.  
I'll keep you updated on this... 😉

Any contributions on this are warmly accepted.

**Reply by draew6** [reply]
@Byloth what about if i am interested in wotlk version?

**Reply by Byloth** [reply]
Well... It would be great, @draew6! 🙃

I need some testers for this feature not merged yet.
It should work, if you simply switch branch and try `feature/bots`! 🤔

If not, it would be a good starting point to understand what's wrong about it.
